### DISCUSSION TITLE
Installazione con Altervista

### DISCUSSION THREAD

**Discussion by scuolagioco** [iisgiua/giuaschool#133] [Richieste di aiuto] [answered]
Salve, provo ad installare il vostro registro ma non riesco. Scorrendo tra le vecchie issue ho visto che molte persone hanno il mio stesso problema con Altervista. Qualcuno potrebbe essermi di aiuto. P.s.  Il mio istituto è senza registro da settembre 🙁 @trinko

**Comment by trinko** [comment, marked as answer]
Non posso aiutarti sull'hosting di Altervista che non conosco.

Posso solo dirti di provare a seguire le indicazioni che ho messo nella [documentazione](https://trinko.github.io/giuaschool-docs/) 
che cercherò di completare pian piano (tempo permettendo).

Ti suggerisco però di provare a iniziare l'installazione e chiedere specificando nel dettaglio cosa non funziona o non riesci a fare: così è più probabile che qualcuno ti risponda.

**Comment by manuelbarberi** [comment]
Guarda i forum precedenti. Ti spiego anche qui come installare il registro:
1) Importa da altervista tutti i file (consiglio di spostarli sulla directory principale) dopo aver modificato il parametro password dal file .env
2) Cambia la versione php a 8.0
3) Vai sulla cartella /public/install/
4) Immetti tutti i parametri del registro 
5) Immetti la password amministratore e il nome utente
6) Accedi attraverso l'url https://nomesito.altervista.org/public
7) Immetti le credenziali e inizia a settare i parametri della scuola 

Se Hai qualche problema contattami attraverso mail : manuelbarberi.mb@libero.it
